templates/part.card.php

```diff
@@ -123,7 +123,7 @@
 				  ng-model="status.edit.description"
 				  ng-change="cardEditDescriptionChanged(); updateMarkdown(status.edit.description)"
 				  autofocus-on-insert> </textarea>
-		<div class="container" ng-click="clickCardDescription($event)"
+		<div class="container" ng-dblclick="clickCardDescription($event)"
```

Comment:
This would be a rather noticeable change in terms of how to edit the description so I think we should have some way to make it clear to the user how to edit the card description.

Reply:
I agree with you. 
Initially this should be a separate issue and pull request. I tried to separate it, but i failed. Is it possible to delete the change of this? What is the next step? Should I add another commit  to my repository to add your changes? When I want to create an issue and a commit, do i have to create a new branch to separate things? This were a lot of questions, i hope it is OK to ask here.

Reply:
Yes, you can add commit to the branch your requesting to be pulled. You can even remove commits then force push it. You can create another branch to open a second pull request from as well.